src/App.tsx

```diff
@@ -1,12 +1,15 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Navigate, NavLink, Route, Routes } from 'react-router-dom';
+import classNames from 'classnames';
+import { TabPage } from './components/TabsPage';
```

Comment:
```suggestion
import { TabsPage } from './components/TabsPage';
```
It's better to use the same name with the file

Reply:
check @max-rozzhalovets comment